programmers/5일차/음양-더하기.js

```diff
@@ -0,0 +1,25 @@
+function solution(absolutes,signs) {
+    const result = absolutes.reduce( (prev,cur,index) => {
+    
+        if (signs[index]) {
```

Comment:
```suggestion
    const result = absolutes.reduce( (prev,cur,index) => {
       if (signs[index]) {
```
여기서는 줄바꿈 안해주셔도 됩니다  

정말 잘푸셨어요 ㅎㅎ